packages/ui-web/src/slate/utils.js

```diff
@@ -57,6 +57,7 @@ export const toMessageBlocks = (nodes) =>
     if (n.type === "emoji") return { type: "emoji", emoji: n.emoji };
     if (n.type === "user") return { type: "user", ref: n.ref };
     if (n.type === "channel-link") return { type: "channel-link", ref: n.ref };
+    if (n.type === "horizontal-divider") return { type: n.type };
```

Comment:
I was using 'callout' as a way to find all places I should add code to handle dividers. guess it's not that straightforward with void elements 😅